atecc608a_se.c

```diff
@@ -0,0 +1,226 @@
+#include "atecc608a_se.h"
+
+#include "atca_status.h"
+#include "atca_devtypes.h"
+#include "atca_iface.h"
+#include "atca_command.h"
+#include "atca_basic.h"
+#include "atca_helpers.h"
+
+/* Uncomment to print results on success */
+//#define DEBUG_PRINT
+
+#ifdef DEBUG_PRINT
+#include <stdio.h>
+#endif
+#include <stdbool.h>
+
+#define ATCAB_INIT()                                        \
+    do                                                      \
+    {                                                       \
+        if (atcab_init(&atca_iface_config) != ATCA_SUCCESS) \
+        {                                                   \
+            status = PSA_ERROR_HARDWARE_FAILURE;            \
+            goto exit;                                      \
+        }                                                   \
+    } while(0)
+
+#define ATCAB_DEINIT()    \
+    do                    \
+    {                     \
+        atcab_release();  \
+    } while(0)
+
+#define ASSERT_STATUS(expression, expected, psa_error)          \
```

Comment:
Please add a comment that explains briefly what the macro does and how to use it.